### ISSUE TITLE
Make filter (hidden/fixed) more visible than in the search box

### ISSUE THREAD

**Issue by steven-r** [exceptionless/Exceptionless.UI#6]
With 2.0, filters are moved to the top bar as part of the search button. 

I was expecting them either as a separate item (have a filter icon next to the search icon) or to have them directly at the tables (as it was in 1.x AFAIR).

This becomes even more important as I want to clear the search term but leave the filters intact (most of the time).

Just one more thought: The time-range filter is a separate element by itself as well; this might be a justification to go with the filters the same way...

**Comment by niemyjski** [comment]
Here is a reference image
![image](https://cloud.githubusercontent.com/assets/1020579/6630236/97c9e7e6-c8e4-11e4-85ff-2131b51e1173.png)

**Comment by niemyjski** [comment]
@ejsmith What do you think? I like them where they are because everything is just a filter.

**Comment by mallapurbharat** [comment]
Had hidden some events, and then had a helluva time trying to get them to be visible again!

I don’t know if the method I followed is the right one, but I managed to get these to show again by clicking on the search button (Magnifying lens on top) and then selecting “Include Hidden”. Once the events showed up, I did a “Bulk action” and unhid them! Phew...  :)

The magnifying lens on the top of the page with the “Show Hidden” option ISN'T really that intuitive. 

It’d be better to have a button saying “Event Filter Options” or something like that at the top of the listbox (or whatever) where you normally show the events. This could have a “Unhide all events” option? 

Also, the calendar icon at the top of the page is equally “unvisible” (sic) and non-intuitive. Why not keep this too at the event list place, with a “Show events from...” title or something?
Hope this makes sense! :)
